### DISCUSSION TITLE
How not to clear the output on quit?

### DISCUSSION THREAD

**Discussion by zer09** [sharkdp/bat#2404] [Q&A] [answered]
Example I open my bashrc ```bat .bashrc``` now when I press ```q``` to quite the output is also cleared, how to prevent that?

**Comment by deadnews** [comment, marked as answer]
https://github.com/sharkdp/bat/blob/master/README.md#automatic-paging

> ### Automatic paging
> 
> 
> By default, `bat` pipes its own output to a pager (e.g. `less`) if the output is too large for one screen.
> If you would rather `bat` work like `cat` all the time (never page output), you can set `--paging=never` as an option, either on the command line or in your configuration file.
> If you intend to alias `cat` to `bat` in your shell configuration, you can use `alias cat='bat --paging=never'` to preserve the default behavior.

For example:
```sh
bat --paging=never .bashrc
```
or via [config file](https://github.com/DeadNews/.dotfiles/blob/main/bat/config); [onfiguration file wiki](https://github.com/sharkdp/bat#configuration-file).
